src/plugins/discover/public/application/main/components/layout/discover_layout.test.tsx

```diff
@@ -39,7 +39,11 @@ import { DiscoverServices } from '../../../../build_services';
 
 setHeaderActionMenuMounter(jest.fn());
 
-function mountComponent(indexPattern: DataView, prevSidebarClosed?: boolean) {
+function mountComponent(
+  indexPattern: DataView,
+  prevSidebarClosed?: boolean,
+  mountOptions: { attachTo?: HTMLElement } = {}
```

Comment:
`toHaveFocus` doesn't work without the component being mounted to an element in the body, so I added `mountOptions` to pass in a mount point.

- https://github.com/enzymejs/enzyme/issues/2337
- https://github.com/jsdom/jsdom/issues/2924